.eslintignore

```diff
@@ -3,3 +3,9 @@
 **/staticBundle.js
 **/main.js
 Libraries/vendor/**/*
+RNTester/**/*
```

Comment:
Nope its not. I figured it would be pointless to check that folder as we don't write most of the sample code in there.